precompiles/staking/abi.json

```diff
@@ -160,31 +160,6 @@
     "name": "Redelegate",
     "type": "event"
   },
-  {
-    "anonymous": false,
```

Comment:
did we remove this?

Reply:
was using an outdated Staking interface as base, it's fixed now.